operators/multiclusterobservability/manifests/base/multicluster-observability-addon/cluster_management_addon.yaml

```diff
@@ -35,7 +35,7 @@ spec:
       - name: global
         namespace: open-cluster-management-global-set
         configs:
-          - group: logging.openshift.io
+          - group: observability.openshift.io
```

Comment:
Is this change from upstream, or does it distinguish MCOA CR's as separate to the CLF installed CR's?

Reply:
With Logging Operator 6.0 the logging team deprecated the old CRDs and created a new one, it's still called `ClusterLogForwarder` but they belong to a different group.

TL;DR: Upstream, they are still the CR's managed by CLO